code/lib/preview-api/src/modules/preview-web/PreviewWeb.test.ts

```diff
@@ -180,6 +182,15 @@ describe('PreviewWeb', () => {
       expect(preview.storyStore.globals!.get()).toEqual({ a: 'c' });
     });
 
+    it('emits the PROJECT_PREPARED event', async () => {
+      await createAndRenderPreview();
+
+      expect(mockChannel.emit).toHaveBeenCalledWith(PROJECT_PREPARED, {
+        argTypes: {},
+        parameters: { docs: expect.any(Object) },
```

Comment:
should we turn this into an inline snapshot instead, or does the content change too much over time?

Reply:
I'm inclined to go the other way in these tests TBH, they are already really long (check out this file, it has 3.5k-ish lines). In some ways the content of the event doesn't really matter -- obviously it shouldn't be weird but it's unlikely to be anything but the right thing, I would have thought.

Reply:
I agree with @tmeasday on this.